core/mondoo-github-security.mql.yaml

```diff
@@ -12,49 +12,67 @@ policies:
       desc: |
         ## Overview
 
-        The GitHub Organization Security policy bundle provides guidance for establishing minimum recommended security and operational best practices for GitHub organizations.
+        GitHub Organization Security by Mondoo provides guidance for establishing minimum recommended security and operational best practices for GitHub organizations.
```

Comment:
We need to bump the version for this policy to 1.4.0

Reply:
I will bump the version. thanks.